libraries-5/src/test/java/com/baeldung/githubapi/UsersUnitTest.java

```diff
@@ -0,0 +1,32 @@
+package com.baeldung.githubapi;
+
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+import org.kohsuke.github.GHMyself;
+import org.kohsuke.github.GHUser;
+import org.kohsuke.github.GitHub;
+
+import java.io.IOException;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class UsersUnitTest {
+    @Test
+    @Disabled
+    void whenWeAccessMyself_WeCanQueryUserDetails() throws IOException {
+        GitHub gitHub = GitHub.connect();
+
+        GHMyself myself = gitHub.getMyself();
+        assertEquals("someone", myself.getLogin());
+        assertEquals("someone@example.com", myself.getEmail());
+        assertEquals(50, myself.getFollows().size());
```

Comment:
Confused about this could ever have passed?

Reply:
It wouldn't. This was just an example. It's not really possible to have stable assertions for this test, since it relies on knowing which user is authenticating.
I can just remove it if that's preferred? Or would it be better to just change the assertions to logs?